### PR TITLE
Disable localization for vs17.10

### PR DESCRIPTION
**Pull request by AR-May**
Disable localization for vs17.10

### CODE REVIEW

File: .vsts-dotnet.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ variables:
     - name: SourceBranch
       value: ''
   - name: EnableReleaseOneLocBuild
-    value: true
+    value: false
```
Comment (rainersigwald) [inline .vsts-dotnet.yml:37]: We'll need to be super careful on the merge-forward to leave this on in the later branches. I wonder if we can add a comment to this line to cause a merge conflict?

**Review by rainersigwald — APPROVED** [review]
LGTM though I would by policy recommend taking the version bump (instead of canceling the auto-scheduled build).

**Review by f-alizada — APPROVED** [review]

File: .vsts-dotnet.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ variables:
     - name: SourceBranch
       value: ''
   - name: EnableReleaseOneLocBuild
-    value: true
+    value: false # Do NOT merge this line to vs17.11
```
Comment (rainersigwald) [inline .vsts-dotnet.yml:37]: I was thinking something like 

```suggestion
    value: false # Disable loc for vs17.10 now that it's shipped
```

and where it's the current branch name, and update it everywhere, then it'll conflict when we hit

```diff
- # Disable loc for vs17.10 now that it's shipped
+ # Enable loc for vs17.11
```

I think?

Comment (AR-May) [reply .vsts-dotnet.yml:37]: We will need to update the version in this comment during the creating the new release branch to make it conflict later. But we can a comment have in main:
# Enable loc for main
where during the final branding PR or later in other PR we update just a comment for future conflict.

Comment (AR-May) [reply .vsts-dotnet.yml]: I will write it in our release instruction

**Pull request merged**